org.flightgear.FlightGear.appdata.xml

```diff
@@ -5,68 +5,69 @@ BugReportURL: https://sourceforge.net/p/fgrun/support-requests/9/
-		<p>El paisaje de FlightGear cubre el mundo entero y se descarga mientras vuela. Visite cualquiera de los 20.000 aeropuertos con una representación precisa de los edificios del aeropuerto en muchos de los mayores aeropuertos internacionales.</p>
-		<p>Más populares hardware de control de vuelo, como yokes y sticks son compatibles. Varios monitores de configuración y multijugador también se destaca.</p>
-        </description>
-	<description xml:lang="it">
-		<p>FlightGear permette di controllare più di 400 velivoli, piccoli e grandi in diverse situazioni, tipo di meteo, stagioni, ciclo diurno e notturno. Questo include velivoli ad elica monomotore, velivoli ad elica monomotore, grandi navi passeggeri a 4 motori, velivoli sperimentali, velivoli ed elicotteri classici e vintage.</p>
-		<p>Il paesaggio di FlightGear copre tutto il mondo e viene scaricato mentre voli. Visita uno qualsiasi dei 20.000 aeroporti con una rappresentazione accurata degli edifici aeroportuali su molti dei più grandi aeroporti internazionali.</p>
-		<p>Sono supportati i più diffusi hardware di controllo del volo, come gioghi e levette. Sono presenti anche impostazioni per monitor multipli e multiplayer.</p>
-	</description>
-	<url type="homepage">https://www.flightgear.org/</url>
-	<screenshots>
-		<screenshot type="default">
-			<image>http://wiki.flightgear.org/images/b/b3/SOTM-Feb18.jpg</image>
-		</screenshot>
-		<screenshot>
-			<image>http://wiki.flightgear.org/images/3/37/SOTY-2017.jpeg</image>
-		</screenshot>
-		<screenshot>
-			<image>http://wiki.flightgear.org/images/2/27/SOTM-Oct17.png</image>
-		</screenshot>
-		<screenshot>
-			<image>http://wiki.flightgear.org/images/2/22/C172p-preview2.jpg</image>
-		</screenshot>
-		<screenshot>
-			<image>http://wiki.flightgear.org/images/7/7f/EC135_Sea_of_fog.jpg</image>
-		</screenshot>
-		<screenshot>
-			<image>http://wiki.flightgear.org/images/4/42/Hot_Stuff_with_H130_at_Stromboli.jpg</image>
-		</screenshot>
-	</screenshots>
-	<content_rating type="oars-1.1">
-		<content_attribute id="social-chat">intense</content_attribute>
-	</content_rating>
-	<project_license>GPL-2.0</project_license>
-	<developer_name>FlightGear Developers</developer_name>
-	<url type="bugtracker">https://sourceforge.net/p/flightgear/codetickets/</url>
-	<url type="donation">https://store.flightgear.org/</url>
-	<url type="help">https://flightgear.sourceforge.net/getstart-en/getstart-en.html</url>
-	<url type="translate">https://wiki.flightgear.org/Howto:Translate_FlightGear</url>
-	<releases>
-		<!-- When updating flightgear, add a new line below (don't remove the older ones) and sort them newest-first -->
-		<release date="2020-12-01" version="2020.3.4" />
-		<release date="2020-11-23" version="2020.3.3" />
-		<release date="2020-05-26" version="2020.1.3" />
-		<release date="2019-01-29" version="2018.3.2" />
-	</releases>
+    <id>org.flightgear.FlightGear</id>
```

Comment:
Please avoid reindenting in a different style, it makes the diff horrible.

Reply:
gnome-builder have strange way of configure tabsize...

however, making excuse is no help

is better now

Reply:
IIRC, Builder uses the same tab style/size as the rest of the document when you press Enter at the end of the line. Anyway it's fixed so all good :)